src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -125,11 +127,6 @@ private int convertToInt(String beingconverted) {
       System.err.println("Could not convert to int: " + beingconverted);
       return 1;
```

Comment:
Here too - consider throwing an IllegalArgumentException.

Reply:
I was thinking you could do something more like adding throw IllegalArgumentException in the function signature, and then doing a catch block around #convertToInt on 83=84:
https://github.com/googleinterns/step64-2020/blob/master/src/main/java/com/google/sps/servlets/YoutubeApi.java#L60

Then in the catch block, you can set defaults currentPage and postPerPage.